src/components/sliders/draw.js

```diff
@@ -71,14 +71,12 @@ module.exports = function draw(gd) {
 
         computeLabelSteps(sliderOpts);
 
-        if(!sliderOpts._commandObserver) {
-            sliderOpts._commandObserver = Plots.createCommandObserver(gd, sliderOpts.steps, function(data) {
-                if(sliderOpts.active === data.index) return;
-                if(sliderOpts._dragging) return;
+        Plots.createCommandObserver(gd, sliderOpts, sliderOpts.steps, function(data) {
```

Comment:
Sounds good. I was going to go with `createOrUpdateCommandObserver`, but that seemed excessive. Changing now.